### PR TITLE
add setters for concurrency props

### PR DESCRIPTION
**Pull request by MoshikEilon**

### CODE REVIEW

File: dist/consumer.js [hunk 1]
```diff
@@ -111,6 +111,15 @@ class Consumer extends events_1.EventEmitter {
         debug('Stopping consumer');
         this.stopped = true;
     }
+    setBatchSize(newBatchSize) {
+        this.batchSize = newBatchSize;
+    }
+    setConcurrencyLimit(newConcurrencyLimit) {
+        const concurrencyLimitDiff = newConcurrencyLimit - this.concurrencyLimit;
+        const newFreeConcurrentSlots = Math.max(0, this.freeConcurrentSlots + concurrencyLimitDiff);
```
Comment (shiraWeiss) [inline dist/consumer.js:125]: 💣

File: dist/consumer.js [hunk 2]
```diff
@@ -282,6 +288,9 @@ class Consumer extends events_1.EventEmitter {
                 .then(this.handleSqsResponse)
                 .catch((err) => {
                 this.emit('error', err);
+                if (isNonExistentQueueError(err)) {
+                    throw new Error(`Could not receive messages - non existent queue - ${this.queueUrl}`);
```
Comment (shiraWeiss) [inline dist/consumer.js:292]: note to self - maybe I should add try catch to my service for these cases?

**Review by shiraWeiss — APPROVED** [review]
looks great 💯

File: dist/consumer.js (hunk 2 above)
Comment (MoshikEilon) [reply dist/consumer.js:292]: mm you don't have the queue names there so i don't think you can perform such validation

**Pull request merged**